src/App.js

```diff
@@ -1,16 +1,37 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import Clock from "./Clock.js";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
+          <Container className="Clock-display">
+            <Row className="Area-clock">
```

Comment:
as shown in class, it's possible to use `.map` to reduce repetition of code. you can also extract these rows into their own component to make that mapping easier